### DISCUSSION TITLE
Translate SimpleLogin mobile app to other languages

### DISCUSSION THREAD

**Discussion by nguyenkims** [simple-login/app#416] [Feature Requests]
Options: POEditor, Weblate

Moved from https://trello.com/c/UsmxGeQB/56-translate-simplelogin-to-other-languages

**Comment by bionicworx** [comment]
Could provide localization in Arabic & possibly Spanish. POEditor would be my platform of choice for this!

**Comment by froozeify** [comment]
I can help for the french localization, I've done translation for multiple project (like Waze) over transifex.com.

POEditor seems to have free unlimited tier for open source project, I think Weblate is limited to 10k string for the free open source tier.

A great feature on transifex is having the possibility to add a screenshot with the string to translate, so translator see the context. I hope the other platform have that too because it's a huge plus.

**Comment by comradekingu** [comment]
Happy to help if you use Weblate.

@froozeify Weblate doesn't have any limits, because it isn't a closed source only-as-a-service platform.
If you mean https://hosted.weblate.org/hosting/ then there aren't any non-theoretical limits there.

On POEditor you have to apply to exceed a limit, and CODEWHALE can change the terms of service at any time.
The tool is too simple.

Transifex is a horrendous mess.

**Comment by dijitalcozum** [comment]
Can you add Turkish language 🙏
